ember-headless-form/rollup.config.mjs

```diff
@@ -24,7 +24,7 @@ export default {
     // These are the modules that should get reexported into the traditional
     // "app" tree. Things in here should also be in publicEntrypoints above, but
     // not everything in publicEntrypoints necessarily needs to go here.
-    addon.appReexports(['components/**/*.js']),
+    addon.appReexports(['components/*.js']),
```

Comment:
this ended up working?

Reply:
Yeah, publicEntrypoints still has the fully `**` glob, for the reason discussed yesterday. This just amkes sure our app re-exportsin `package.json` only exposes the root component to the app, when doing the classic string lookups fomr DI rather than imports:

<img width="644" alt="image" src="https://user-images.githubusercontent.com/1325249/213222017-d96472f1-09ff-4638-a1cd-96f6fe4924df.png">

So this is unrelated to the Glint issue we discussed, if this was your concern?

Reply:
ahh! yup, I was confused. thanks!